### PR TITLE
Fix jump to view page

### PR DESCRIPTION
**Pull request by morpheus-87**
This PR fixes the jump to the view page after successful saving and updating by
* returning the saved object with a http status code
* constructing the view url in the editor

### CODE REVIEW

File: dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/ContentTreesController.java [hunk 1]
```diff
@@ -96,31 +94,25 @@ public String list(
   @PostMapping("/api/contenttrees/new")
   public ResponseEntity save(@RequestBody ContentTree contentTree)
       throws IdentifiableServiceException {
-    ContentTree contentTreeDb = null;
-    HttpHeaders headers = new HttpHeaders();
     try {
-      contentTreeDb = service.save(contentTree);
-      headers.setLocation(URI.create("/contenttrees/" + contentTreeDb.getUuid().toString()));
+      ContentTree contentTreeDb = service.save(contentTree);
+      return ResponseEntity.ok(contentTreeDb);
```
Comment (jbaiter) [inline dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/ContentTreesController.java]: nit: `.created(...)` wäre semantisch besser

Comment (morpheus-87) [reply dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/ContentTreesController.java]: Ok, macht Sinn, aber wie ist es beim Update?

Comment (jbaiter) [reply dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/ContentTreesController.java]: da ist `.ok()` in ordnung :-) https://github.com/for-GET/http-decision-diagram

Comment (morpheus-87) [reply dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/ContentTreesController.java]: :+1:

**Pull request merged**